src/Core/src/Handlers/SearchBar/SearchBarHandler.iOS.cs

```diff
@@ -32,9 +32,12 @@ protected override void ConnectHandler(MauiSearchBar nativeView)
 		{
 			nativeView.CancelButtonClicked += OnCancelClicked;
 			nativeView.SearchButtonClicked += OnSearchButtonClicked;
-			nativeView.TextPropertySet += OnTextPropertySet;
+			nativeView.TextChanged += OnTextChanged;
```

Comment:
I think the event we need to use is similar to what we have setup for EditorHandler/EntryHandler

https://github.com/dotnet/maui/blob/b09979d93ac544d4b0dd9fd0df5c25da7e91392c/src/Core/src/Platform/iOS/MauiTextView.cs#L29